images/kube-proxy/OWNERS

```diff
@@ -0,0 +1,11 @@
+reviewers:
```

Comment:
drop this file; we're going with just a single top-level `OWNERS` file

Reply:
ok